cpp/oneapi/dal/backend/primitives/select.cpp

```diff
@@ -0,0 +1,249 @@
+/*******************************************************************************
```

Comment:
Incorrect naming of the file. All `.cpp` files that should be compiled only by DPC++ compiler should end with `_dpc`